src/khoj/database/models/__init__.py

```diff
@@ -325,6 +326,12 @@ class EntrySource(models.TextChoices):
     corpus_id = models.UUIDField(default=uuid.uuid4, editable=False)
 
 
+class FileObject(BaseModel):
```

Comment:
Let's store the `user` on the `FileObject`.

```suggestion
class FileObject(BaseModel):
    # Same as Entry but raw will be a much larger string
    file_name = models.CharField(max_length=400, default=None, null=True, blank=True)
    raw_text = models.TextField()
    user = models.ForeignKey(KhojUser, on_delete=models.CASCADE)
```